README.rst

```diff
@@ -7,6 +7,8 @@ your own applications \* Reproducibly train models from a variety of
 frameworks on big & small data, without worrying about installing
 dependencies
 
+This repository requires use of Python 3.
```

Comment:
We could be a bit more specific, e.g:

"We recommend calling the apps in this library from a Python 3 environment. The apps run in Python 3 conda environments, so it may not be possible to load the models produced by the apps back into Python 2 environments."